code/datums/components/defibrillator.dm

```diff
@@ -330,8 +338,31 @@
 	add_attack_logs(user, target, "Stunned with [parent]")
 	set_cooldown()
 	busy = FALSE
-	return
 
+/datum/component/defib/proc/combat_fibrillate(mob/user, mob/living/carbon/human/target)
+	if(!istype(target))
+		return
+	busy = TRUE
+	target.adjustStaminaLoss(60)
+	target.emote("gasp")
+	to_chat(target, "<span class='danger'>[user] touches [target] lightly with [parent]!</span>")
+	add_attack_logs(user, target, "Stunned with [parent]")
+	if(!IS_HORIZONTAL(target))
+		target.KnockDown(4 SECONDS)
+		set_short_cooldown()
+		busy = FALSE
+		return
+	if(do_after(user, 1.5 SECONDS, target = target))
+		target.set_heartattack(TRUE)
```

Comment:
Probably want another admin log here, and there should probably be a visible message before the doafter, similar to how normal defib application works